server/src/test/java/io/crate/integrationtests/PgCatalogITest.java

```diff
@@ -457,4 +457,12 @@ public void test_named_primary_key_constraint_names_are_visible() {
         execute("select conname from pg_catalog.pg_constraint where conname = 'c_aa'");
         assertThat(response).hasRows("c_aa");
     }
+
+    @Test
+    public void test_renamed_columns_are_reflected_on_pg_attribute() {
```

Comment:
Could merge both `test_renamed_columns_are_reflected_on_pg_attribute` and `test_renamed_columns_are_reflected_on_information_schema` into a single `test_renamed_column_is_reflected_in_sys_tables`. Maybe even make it part of a one of the `AlterTableRenameColumnIntegrationTest` cases. It's all the same user scenario: Rename column must change the column name.